source/stylesheets/site/_utilities.scss

```diff
@@ -0,0 +1,5 @@
+.u-hide-small-screen {
```

Comment:
Just utility classes. I'll document them once we have a README, which I'll get started on today.